src/Image.php

```diff
@@ -79,12 +64,7 @@ public function useImageDriver(string $imageDriver)
         return $this;
     }
 
-    /**
-     * @param callable|Manipulations $manipulations
-     *
-     * @return $this
```

Comment:
It might be beneficial to retain this line as it documents that the *same* object is returned rather than a new one of the same class. Same comment applies to quite a few places.